knowledge_graphs/validators/parameter_inspector.py

```
@@ -141,55 +131,24 @@
 
     # Handle special parameters
     if param_str == "self":
-        return {
-            "name": "self",
-            "has_default": False,
-            "is_special": True,
-            "param_type": "instance_method"
-        }
+        return {"name": "self", "has_default": False, "is_special": True, "param_type": "instance_method"}
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)